### PR TITLE
GPM: use local files instead of remote manifests.

### LINKED ISSUE #21: Impossibile to use gpm module in environment with constrained Internet access
I am not able to deploy the `gpm` module in environment with constrained Internet access since we're pulling the Kustomize base directly from GitHub.

I know that it will lead to some duplications, but can we consider to just add the manifests to this repository as well instead of pulling the base from GitHub?

Thank you.

### PR DESCRIPTION
**Pull request by ralgozino**
This PR takes the manifests for the GPM package from upstream and renders them local to fix #21.

Adds also a `MAINTENANCE.md` file with instructions.

### CODE REVIEW

File: katalog/gatekeeper/gpm/MAINTENANCE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Gatekeeper Policy Manager Package Maintenance
+
+To maintain the GPM package, you'll need to keep updated with upstream the following files:
```
Comment (nutellinoit) [inline katalog/gatekeeper/gpm/MAINTENANCE.md:3]: Let's add the link for the upstream project somewhere

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Minor comment for the `MAINTENANCE.md` file, other than that LGTM 🚀

File: katalog/gatekeeper/gpm/MAINTENANCE.md (hunk 1 above)
Comment (ralgozino) [reply katalog/gatekeeper/gpm/MAINTENANCE.md:3]: it is as a comment in each YAML file (it is written a little below at line 10) and the upstream project is linked in the readme also. Is this what you mean or something else?

Comment (nutellinoit) [inline katalog/gatekeeper/gpm/MAINTENANCE.md:3]:
```suggestion
To maintain the GPM package, you'll need to keep updated with upstream the following files (check the main `README.md` for the upstream prioject link):
```

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Left a suggestion for `MAINTENANCE.md`

**Review by nutellinoit — APPROVED** [review]
LGTM 🚀

**Pull request merged**